### ISSUE TITLE
Encourage console warning for method normalization

### ISSUE THREAD

**Issue by annevk** [whatwg/fetch#259]
If we log a warning whenever https://fetch.spec.whatwg.org/#concept-method-normalize is invoked, perhaps that'll teach folks that HTTP method names are case-sensitive.

@wanderview @tyoshino any interest in adding such a warning? You probably only want to do it once per global or some such, to avoid spamming too much.

**Comment by tyoshino** [comment]
This algorithm runs only for input from script via `init`, right? Then, once people see a warning, they can respond to it just by fixing their code.

and, yeah, we can choose to show it only once.

I think it's good.

I'm also interested in showing a warning when a Headers object rejects append of some header silently due to the guard set to it. Should I file another issue for it?

**Comment by annevk** [comment]
We can do both in this issue. I just need to figure out what kind of wording HTML uses to encourage reporting things to the console and then I'll update both places.

**Comment by annevk** [comment]
Trying to think this through again, I think this would end up being too noisy so implementations should probably not issue a warning here. This will have to be a quirk developers discover on their own. (Happy to discuss further though in follow-up comments.)
